docs/collecting-data/collecting-from-own-applications/mobile-trackers/configuring-how-events-are-sent/index.md

```diff
@@ -0,0 +1,157 @@
+
+<Tabs groupId="platform">
+  <TabItem value="ios" label="iOS" default>
+
+```swift
+let eventStore = SQLiteEventStore(namespace: kNamespace);
+let emitterConfig = EmitterConfiguration()
+      .eventStore(eventStore)
+```
+
+  </TabItem>
+  <TabItem value="android" label="Android">
+
+```java
+EventStore eventStore = new SQLiteEventStore(getApplicationContext(), kNamespace);
+
+EmitterConfiguration emitterConfiguration = new EmitterConfiguration()
+      .eventStore(eventStore);
+```
+
+  </TabItem>
+</Tabs>
+
+In the example above we used the `SQLiteEventStore` but it can be used any custom component that implements the `EventStore` interface.
+
+## What happens if an event fails to send?
+
+To prevent unnecessary requests being made while the collector is unavailable, a backoff is added to all subsequent event sending attempts. This resets after a request is successful.
+
+The tracker has an option for setting response codes not to retry after. The intended use is for codes such as `400 Bad Request`, `401 Unauthorised`, `403 Forbidden`, `410 Gone`, `422 Unprocessable Entity`. When received in response from the Collector, the tracker doesn't retry to send events. Requests with all other 3xx, 4xx, and 5xx status codes are retried. The set of status codes for which events should be retried or not is customizable in `EmitterConfiguration`.
+
+By default, the tracker retries sending events on all 3xx, 4xx, and 5xx status codes except the status codes indicated above. You may override the default behaviour using the `customRetryForStatusCodes`. Please note that not retrying sending events to the Collector means that the events will be dropped when they fail to be sent. The `customRetryForStatusCodes` needs a dictionary that maps integers (status codes) to booleans (true for retry and false for not retry).
+
+<Tabs groupId="platform">
+  <TabItem value="ios" label="iOS" default>
+
+```swift
+let emitterConfig = EmitterConfiguration()
+    .customRetryForStatusCodes({403, true}) // retry sending events even if collector returns 403 status
+```
+
+  </TabItem>
+  <TabItem value="android" label="Android">
+
+```java
+EmitterConfiguration emitterConfiguration = new EmitterConfiguration()
+      .customRetryForStatusCodes(new HashMap<>() {{ put(403, true); }});
+```
+
+  </TabItem>
+</Tabs>
+
+## Configuring how many events to send in one request
+
+The tracker sends events in batches. The tracker allows only a choice of 1 (`BufferOption.single`), 10 (`BufferOption.defaultGroup`) or 25 (`BufferOption.largeGroup`) events per payload. The tracker sends the events as soon as possible using the `BufferOption.Single` option. Even with a different batching option the events are sent as soon as the event sending is automatically triggered after 5 seconds.
+
+<!-- (TODO: This is confusing and terrible, not clearly documented and should be coherent with the other trackers). -->
```

Comment:
I think we can remove this, I'll create a Jira task to eventually fix this in the future.